generators/client/files-angular.js

```diff
@@ -150,7 +150,7 @@ const files = {
             ]
         },
         {
-            condition: generator => !generator.skipServer,
+            condition: generator => generator.skipServer,
```

Comment:
This will only generate the file when skipServer is true.  I believe you always want to copy the file, so it should be moved [up to here with the other interceptors](https://github.com/clarenced/generator-jhipster/blob/92c600803e67f45ed64c12d33379d55546da9abf/generators/client/files-angular.js#L142)

Reply:
OK

Reply:
@ruddell done